homeassistant/components/alarmdecoder/manifest.json

```diff
@@ -3,8 +3,9 @@
   "name": "AlarmDecoder",
   "codeowners": [],
   "config_flow": true,
+  "integration_type": "device",
   "documentation": "https://www.home-assistant.io/integrations/alarmdecoder",
   "iot_class": "local_push",
   "loggers": ["adext", "alarmdecoder"],
-  "requirements": ["adext==0.4.2"]
+  "requirements": ["adext==0.4.3"]
```

Comment:
It is usually recommended that dependency bumps are done as standalone PR.
Only the changes directly linked to the dependency bump (for example when a property has been renamed) should go in the dependency bump PR.

Reply:
Thanks @epenet for taking the time to review my change.

Yes I saw that in the instructions, but wasn't sure this one made sense.  Curious to hear your feedback about this one and whether it's the right way to go.

Moreover, adext now exposes a new API to async/wait when it's done initializing.  The AlarmDecoder extension now wait for the object to be created before initializing.  The only way I can think to slice it would be to do the API bump and add the API async/wait call in AlarmDecoder and do no other change.  In isolation, that change would seem rather odd because there's currently no reason to wait for the initialization to happen given the old implementation.  It only really makes sense in light that I need now information from the initialization in order to create the entity in the extension.

If you believe that's still the way to go, I'd be happy to make the change accordingly, but it didn't quite obvious that it was the intention of the rule vs a simpler case of API bump where the API changes with no actual net effect on the code for example.

Please advise.

Reply:
It's still good practise, you can indicate that the bump is "just to keep to up-to-date", or that it is "preliminary for follow-up PRs that will make use of the functionnality".

Reply:
Ok no problem at all.  Here's the relevant PR that I submitted to handle the bump only:

[https://github.com/home-assistant/core/pull/117496](https://github.com/home-assistant/core/pull/117496)